requirements/optional.txt

```diff
@@ -1,10 +1,11 @@
 SimpleITK
 astropy>=3.1.2
-qtpy
-pyamg
-dask[array]>=1.0.0,!=2.17.0
 # cloudpickle is necessary to provide the 'processes' scheduler for dask
 cloudpickle>=0.2.1
+dask[array]>=1.0.0,!=2.17.0
+matplotlib>=3.0.3
+pyamg
```

Comment:
I assume you moved the order here to make things alphabetical? If so then this `pyamg` should be below pooch

Reply:
:facepalm: Thank you @grlee77, I will fix that :wink: